src/components/ContentUploader/ContentUploader.js

```diff
@@ -358,6 +367,7 @@ class ContentUploader extends Component<Props, State> {
         onUpload(item.boxFile);
 
         this.updateViewAndCollection(items);
+        this.upload();
```

Comment:
Did you mean to add this here?

Reply:
Yes, this is for kicking off uploads for other pending items.
Upload is started when
- new items are added to the queue
- item upload succeeded
- item upload failed